internal/cmd/liquid.go

```diff
@@ -0,0 +1,416 @@
+		if err != nil {
+			return err
+		}
+	}
+
+	// TODO: If I interpret cmp.Diff correctly, serviceCapacityReport is empty
+	// Example diff:
+	// -                                       Capacity:      0,
+	// +                                       Capacity:      197156,
+	// -                                       Usage:         &0,
+	// +                                       Usage:         &75774,
+	if compare {
+		diff := cmp.Diff(serviceCapacityReport, localServiceCapacityReport)
+		if diff == "" {
+			fmt.Println("ServiceCapacityReports are identical")
+		} else {
+			fmt.Println(diff)
+		}
+	}
+
+	return nil
+}
+
+///////////////////////////////////////////////////////////////////////////////
+// LIQUID Report Usage
+
+type liquidReportUsageCmd struct {
+	*cobra.Command
+
+	liquidOperationFlags
+}
+
+func newLiquidReportUsageCmd() *liquidReportUsageCmd {
+	liquidReportUsage := &liquidReportUsageCmd{}
+	cmd := &cobra.Command{
+		Use:     "report-usage $SERVICE_TYPE $PROJECT_UUID",
+		Short:   "Get usage data within a project of a liquid",
+		Long:    "Get usage data (as well as applicable quotas) within a project across all resources of a liquid. This command requires a cloud-admin token.",
+		Args:    cobra.ExactArgs(2),
+		PreRunE: authWithLimesAdmin,
+		RunE:    liquidReportUsage.Run,
+	}
+
+	// Flags
+	doNotSortFlags(cmd)
+	liquidReportUsage.liquidOperationFlags.AddToCmd(cmd)
+
+	liquidReportUsage.Command = cmd
+	return liquidReportUsage
+}
+
+func GetLiquidUsageReport(provider *gophercloud.ProviderClient, opts liquidapi.ClientOpts, ctx context.Context, projectID string, serviceUsageRequest *liquid.ServiceUsageRequest, compare bool) (liquid.ServiceUsageReport, error) {
+	liquidClient, err := liquidapi.NewClient(provider, gophercloud.EndpointOpts{}, opts)
+	if err != nil {
+		return liquid.ServiceUsageReport{}, util.WrapError(err, "could not instantiate new LIQUID client")
+	}
+	serviceUsageReport, err := liquidClient.GetUsageReport(ctx, projectID, *serviceUsageRequest)
+	if err != nil {
+		return liquid.ServiceUsageReport{}, util.WrapError(err, "could not fetch service usage report from LIQUID api")
+	}
+	if !compare {
+		fmt.Println(serviceUsageReport)
+	}
+	return serviceUsageReport, nil
+}
+
+func (c *liquidReportUsageCmd) Run(cmd *cobra.Command, args []string) error {
+	serviceType := args[0]
+	projectID := args[1]
+
+	endpoint := c.liquidOperationFlags.endpoint
+	compare := c.liquidOperationFlags.compare
+	if compare && (endpoint == "" || serviceType == "") {
+		return errors.New("argument $SERVICE_TYPE and flag --endpoint are both required for comparison mode")
+	}
+
+	// TODO: "Hacky", this should probably be fixed in the limes api endpoint right?
+	// I.e. Api currently only works when receiving service type cinder and not with liquid-cinder
+	reqServiceType := strings.TrimPrefix(serviceType, "liquid-")
```

Comment:
I would rather only take the plain service type without `liquid-` on the CLI, and then add `liquid-` on our side (i.e. in the GetLiquidUsageReport call).